Makefile

```diff
@@ -45,7 +45,7 @@ assets: $(REACT_APP_OUTPUT_DIR)
 	# Un-setting GOOS and GOARCH here because the generated Go code is always the same,
 	# but the cached object code is incompatible between architectures and OSes (which
 	# breaks cross-building for different combinations on CI in the same container).
-	cd web/ui && GO111MODULE=$(GO111MODULE) GOOS= GOARCH= $(GO) generate -x -v $(GOOPTS)
+	cd web/ui && GO111MODULE=$(GO111MODULE) CI= GOOS= GOARCH= $(GO) generate -x -v $(GOOPTS)
```

Comment:
What does this do for the `go generate` step? Is it needed?

Reply:
not sure, it was just to see if it would fix the test job

Reply:
Ah yeah, let's remove that again then, the `CI` var was only interpreted by the yarn lint / build stuff.

Reply:
or actually I should just add it in the package.json in the build script. WDYT ?

Reply:
No, we still want to keep the CircleCI tests the same, we just want to set `CI=` for Netlify, where we do the replacement.

Reply:
mmm in that case the warnings are an issue because the build job is failing which is the first step in the test job.
So should I fix them then ?